downstairs/src/extent.rs

```diff
@@ -518,17 +546,18 @@ impl Extent {
         &mut self,
         job_id: JobId,
         requests: &[crucible_protocol::ReadRequest],
-    ) -> Result<Vec<crucible_protocol::ReadResponse>, CrucibleError> {
+        out: &mut BytesMut,
```

Comment:
We need to update the comment above this method as there is no more `responses`

Reply:
Yup, fixed in 6c12d1b7944333f9a0a6d686c84ae0ae56e1bb5c